fast_response/AlertFollowup.py

```diff
@@ -266,6 +282,7 @@ class TrackFollowup(AlertFollowup):
     _fix_index = True
     _float_index = not _fix_index
     _index = 2.5
+    _llh_map = True
```

Comment:
Should this be false as well?

Reply:
nope, that's intentional here, I'm using this for labeling the zoom skymap, so this is the only one that should be false